src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -180,6 +180,31 @@ s! {
         b_refcount: ::c_int,
         b_destroying: ::c_int,
     }
+
+    pub struct kinfo_vmentry {
+        pub kve_structsize: ::c_int,
+        pub kve_type: ::c_int,
+        pub kve_start: u64,
+        pub kve_end: u64,
+        pub kve_offset: u64,
+        pub kve_vn_fileid: u64,
+        pub kve_vn_fsid_freebsd11: u32,
+        pub kve_flags: ::c_int,
+        pub kve_resident: ::c_int,
+        pub kve_private_resident: ::c_int,
+        pub kve_protection: ::c_int,
+        pub kve_ref_count: ::c_int,
+        pub kve_shadow_count: ::c_int,
+        pub kve_vn_type: ::c_int,
+        pub kve_vn_size: u64,
+        pub kve_vn_rdev_freebsd11: u32,
+        pub kve_vn_mode: u16,
+        pub kve_status: u16,
+        __kve_type_spec: ::uintptr_t,
+        pub kve_vn_rdev: u64,
+        _kve_is_spare: [::c_char; 8],
+        pub kve_path: [::c_char; ::PATH_MAX as usize],
```

Comment:
Hmm, the definition seems correct but CI complains about its size on FreeBSD 12 and 13. Also, the definition on FreeBSD 11 is a bit different.